requirements.txt

```diff
@@ -0,0 +1 @@
+prometheus_client==0.7.1
```

Comment:
this is not needed? Isn't this for Python?

Reply:
I was not sure about all that requirement file setup, it contained like 20 lines before I removed everything. I can't tell you if this is needed or not yet, let's try to figure that out later 🚀

Reply:
I am 93% sure, it is for Python dependency versioning.